app/controllers/DataSetController.scala

```diff
@@ -117,14 +125,14 @@ class DataSetController @Inject()(val messagesApi: MessagesApi) extends Controll
   }
 
   def updateTeams(dataSetName: String) = SecuredAction.async(parse.json) { implicit request =>
-    withJsonBodyAs[List[String]] { teams =>
+    withJsonBodyAs[List[BSONObjectID]] { teams => //TODO
```

Comment:
is this "TODO" still relevant?

Reply:
Yes, should be still relevant. Using `BSONObjectID` here is inconsistent and brakes wK. When requesting a dataset it has an Attribute `selectedTeams` (`List[String]`). However, when calling this update route the backend expects `List[BSONObjectID]` which is inconsistent. The frontend still sends strings.